spec/dex-oracle/driver_spec.rb

```diff
@@ -20,7 +20,7 @@
     allow_any_instance_of(Driver).to receive(:get_driver_dir).and_return('/data/local')
     Driver.new(device_id)
   end
-  let(:driver_stub) { 'export CLASSPATH=/data/local/od.zip; app_process /system/bin org.cf.oracle.Driver' }
+  let(:driver_stub) { 'cd /data/local; export CLASSPATH=/data/local/od.zip; app_process /system/bin org.cf.oracle.Driver' }
```

Comment:
Is this `cd` necessary? Why did it work before?

Reply:
I think it is necessary as `org.cf.oracle.Driver` does not have a hard-coded `DRIVER_DIR` anymore and it needs to find it out somehow else.